app/ports/templates/ports/stats.html

```diff
@@ -156,14 +156,28 @@
     <hr>
     <br>
     {% csrf_token %}
-    <form class="form-inline justify-content-center" method="get" action="{% url 'stats_home' %}#os-chart">
-        <label for="days"><h5>The charts below show stats for users who made submissions in last: </h5></label>
-        <select onchange="this.form.submit()" class="form-control ml-2" name="days" id="days">
-        {% for i in allowed_days|slice:"1:" %}
-            <option class="format-option" value="{{ i }}" {% if days == i %}selected{% endif %}>{{ i }} Days</option>
-        {% endfor %}
+    <form id="duration-selector" class="form-inline justify-content-center" method="get" action="{% url 'stats_home' %}#duration-selector">
+        <lable for="days">Displaying statistics for users who made submissions in previous:</lable>
```

Comment:
Now this template looks the same as the one in `templates/ports/port-detail/installation_stats.html`. Could we use the exact same template?

Reply:
This is quite different from that one. Charts like, `Users vs Month`, `Versions vs Month` and a lot of other things are different.